core/systemcontracts/upgrade.go

```diff
@@ -383,13 +383,13 @@ func init() {
614a2657fe5b60200101906001600160f81b031916908160001a905350614a4782826148db565b95945050505050565b8051600090614a6157506000611196565b6020820151805160001a9060c0821015614a8057600092505050611196565b5060019392505050565b8051600090811a6080811015614aa4576000915050611196565b60b8811080614abf575060c08110801590614abf575060f881105b15614ace576001915050611196565b60c0811015614ae25760b519019050611196565b60f519019050611196565b80516000908190811a6080811015614b085760019150614c19565b60b8811015614b1d57607e1981019150614c19565b60c0811015614b9757600060b78203600186019550806020036101000a865104915060018101820193505080831015614b91576040805162461bcd60e51b81526020600482015260116024820152706164646974696f6e206f766572666c6f7760781b604482015290519081900360640190fd5b50614c19565b60f8811015614bac5760be1981019150614c19565b600060f78203600186019550806020036101000a865104915060018101820193505080831015614c17576040805162461bcd60e51b81526020600482015260116024820152706164646974696f6e206f766572666c6f7760781b604482015290519081900360640190fd5b505b5092915050565b8051600090601514614c3157600080fd5b612bf78261318d565b60006060614c53846402540be40063ffffffff612f0616565b60408051600480825260a0820190925291955060609190816020015b6060815260200190600190039081614c6f579050509050614c928760ff16612fad565b81600081518110614c9f57fe5b6020026020010181905250614cbc866001600160a01b0316612f8a565b81600181518110614cc957fe5b6020026020010181905250614cdd85612fad565b81600281518110614cea57fe5b6020026020010181905250614d048463ffffffff16612fad565b81600381518110614d1157fe5b60200260200101819052506060614d2782612fc0565b94989497509395505050505050565b80614d4057612e21565b5b60208110614d60578251825260209283019290910190601f1901614d41565b915181516020939093036101000a6000190180199091169216919091179052565b600082614d9057506000612bf7565b82820282848281614d9d57fe5b0414612eff5760405162461bcd60e51b8152600401808060200182810382526021815260200180614ec16021913960400191505060405180910390fd5b60405180606001604052806003906020820280368337509192915050565b6040518060400160405280614e0b614e18565b8152602001600081525090565b60405180604001604052806000815260200160008152509056fe7468652042534352656c61796572466565206d6f642074656e20646563696d616c73206d757374206265207a65726f7468652042534352656c61796572466565206d757374206265206c657373207468616e2072656c61796572466565746865206d696e44656c65676174696f6e206d7573742062652067726561746572207468616e2072656c61796572466565536166654d6174683a206d756c7469706c69636174696f6e206f766572666c6f777468652072656c61796572466565206d6f642074656e20646563696d616c73206d757374206265207a65726f746865206d6573736167652073656e646572206d75737420626520676f7665726e616e636520636f6e74726163747468652072656c61796572466565206d757374206265206c657373207468616e206d696e44656c65676174696f6e7468652072656c61796572466565206d757374206265206d6f7265207468616e2042534352656c61796572466565746865206d6573736167652073656e646572206d7573742062652063726f737320636861696e20636f6e7472616374a26469706673582212202d43db3e2ebde4487e0172cc743868769dac674a6b5ac4e5b38d932f5fe77d2964736f6c63430006040033",
```

Comment:
Why mainNet not changed?

Reply:
We must have some feedback from audit team and patch the smart contract, so no need to set the mainnet byte code in this release.